<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>497</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Tempeartures (separated by comma, 1.8-350 K)</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>B290X</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QLabel" name="lbUnitLimit">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QComboBox" name="cbSource">
       <item>
        <property name="text">
         <string>voltage</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>current</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Limit (compliance)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>s</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="ValLineEdit" name="leStop">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Integration time (8e-6 to 2 s)</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source mode</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QSpinBox" name="sbPoints">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>PPMS</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="ValLineEdit" name="leIntegration">
       <property name="text">
        <string>0.04</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Points (1 to 10000)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="ValLineEdit" name="leStart">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QLabel" name="lbUnitStop">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="ValLineEdit" name="leLimit">
       <property name="text">
        <string>0.05</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QComboBox" name="cbPPMS"/>
     </item>
     <item row="5" column="3">
      <widget class="QLabel" name="lbUnitStart">
       <property name="text">
        <string>V</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="ValLineEdit" name="leTemps">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QComboBox" name="cbB290X"/>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QCheckBox" name="cbFourWire">
       <property name="text">
        <string>Four-wire measurement</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0" colspan="4">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2" colspan="2">
      <widget class="QCheckBox" name="cbUsePpms">
       <property name="text">
        <string>Use PPMS to control temperature</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="4">
      <widget class="QPushButton" name="pbLoad">
       <property name="text">
        <string>Load from file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValLineEdit</class>
   <extends>QLineEdit</extends>
   <header>validator.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cbUsePpms</tabstop>
  <tabstop>cbPPMS</tabstop>
  <tabstop>cbB290X</tabstop>
  <tabstop>cbFourWire</tabstop>
  <tabstop>cbSource</tabstop>
  <tabstop>leStart</tabstop>
  <tabstop>leStop</tabstop>
  <tabstop>sbPoints</tabstop>
  <tabstop>leLimit</tabstop>
  <tabstop>leIntegration</tabstop>
  <tabstop>leTemps</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
